data/DAMIC_2020.csv

```diff
@@ -0,0 +1,23 @@
+mass,cross_section
```

Comment:
could you add a comment-# in front of the headers, I think github then parses it as .csv?

Reply:
Okay, done. This is not present in the other .csv-files (if they include headers).

Reply:
do as I say etc.... ;)